tests/functional/vault/test_config.py

```diff
@@ -43,32 +45,174 @@ def test_vault_name_symbol_override(guardian, gov, rewards, token, Vault):
 
 
 @pytest.mark.parametrize(
-    "getter,setter,val",
+    "getter,setter,val,gov_allowed,mgmt_allowed,guard_allowed",
     [
-        ("name", "setName", "NewName yVault"),
-        ("symbol", "setSymbol", "yvNEW"),
-        ("emergencyShutdown", "setEmergencyShutdown", True),
-        ("guardian", "setGuardian", None),
-        ("rewards", "setRewards", None),
-        ("management", "setManagement", None),
-        ("performanceFee", "setPerformanceFee", 1000),
-        ("managementFee", "setManagementFee", 1000),
-        ("depositLimit", "setDepositLimit", 1000),
+        ("name", "setName", "NewName yVault", True, False, False),
+        ("symbol", "setSymbol", "yvNEW", True, False, False),
+        ("emergencyShutdown", "setEmergencyShutdown", True, True, False, True),
+        ("guardian", "setGuardian", None, True, False, True),
+        ("rewards", "setRewards", None, True, False, False),
+        ("management", "setManagement", None, True, False, False),
+        ("performanceFee", "setPerformanceFee", 1000, True, False, False),
+        ("managementFee", "setManagementFee", 1000, True, False, False),
+        ("depositLimit", "setDepositLimit", 1000, True, False, False),
```

Comment:
Doesn't seem like management is used at all in these methods. Perhaps delete it?

Reply:
Also note that governance is always allowed, so that flag doesn't do anything

Reply:
fixed!